nncf/experimental/tensorflow/patch_tf.py

```diff
@@ -198,12 +197,6 @@ def patch_tf_operations() -> None:
                         curr_module = getattr(curr_module, curr_name)
                     setattr(curr_module, name, tf_op_wrapper)
 
-            # TODO(andrey-churkin): Changes references from the `tensorflow.python.ops.nn`
```

Comment:
@andrey-churkin as I understand this is save to delete now?

Reply:
Yes